lib/Targets/AIEVecToCpp/TranslateAIEVecToCpp.cpp

```diff
@@ -2423,8 +2417,8 @@ static LogicalResult printOperation(CppEmitter &emitter,
           for (OpResult result : op->getResults()) {
             if (failed(emitter.emitVariableDeclaration(
                     result, /*trailingSemicolon=*/true))) {
-              return WalkResult(
-                  op->emitError("unable to declare result variable for op"));
+              return {
+                  op->emitError("unable to declare result variable for op")};
```

Comment:
Avoid repeating the return type from the declaration; use a braced initializer list instead